### PR TITLE
Update CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by m4v15**
Allow for recipes not yet cooked at the guesthouse

### CODE REVIEW

**Comment by edgarfinn** [comment]
❤️ this

👍

**Review by edgarfinn — APPROVED** [review]

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -126,11 +132,17 @@ Make sure your PR has the following
     e.g. anyone else who ate the food, or has expressed an interest in knowing the recipe
     + if your PR _requires_ a review from a particular person / people _before_ it is ready to be merged, specify this within the body of your PR
     + Only maintainers should merge the PR's - if they are slacking, please feel free to give them a nudge in the comments of the PR.
+   6. **If your recipe is yet to be cooked:** Please give the PR a `Pending Cooking` label
+  
 
 ## Instructions for maintainers
 Once a pull request has been reviewed by 3 people **who ate the meal**, as well as or including a maintainer, it can be merged. A PR should never be merged if a reviewer has requested changes and not yet re-reviewed it.
```
Comment (yvonne-liu) [inline CONTRIBUTING.md:139]: 😄

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -1,5 +1,11 @@
 # Contribution guidelines
 
+Although life for Nazareth recipes began as a humble repo, designed to store the recipes of meals cooked at the Nazareth guesthouse, it is growing to become a place to store any communal recipe made by a member of Founders And Coders. We at Nazareth Recipes want to both embrace this bright new future, but also to remember and respect our past.
```
Comment (yvonne-liu) [inline CONTRIBUTING.md]: Inconsistent capitalization. Nazareth Recipes or Nazareth recipes?

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -1,5 +1,11 @@
...
+ 
+ Thus, although only recipes cooked within a Founders And Coders community environment (most likely the Nazareth Guesthouse) will be merged into the repo, we would :heart:love:heart: to have more recipes avaliable and to encourage wider participation in this project.
+ 
+ So if you have a recipe that you think deserves to be on here, but you are unable to cook it at a communal event, please follow the exact same steps as below, but when making a pull request, the PR should have the `Pending cooking` label attached to it. Once it has been cooked and eaten by members of the community, those lucky diners can then review and merge!*
```
Comment (yvonne-liu) [inline CONTRIBUTING.md]: `Pending cooking` or `pending cooking`? Worth mentioning that there are inconsistent capitalization in the labels as well.

Comment (matthewdking) [reply CONTRIBUTING.md]: *capitalisation

File: CONTRIBUTING.md (hunk 2 above)
Comment (matthewdking) [reply CONTRIBUTING.md]: *capitalisation

**Review by finnhodgkin — APPROVED** [review]
Do you think we need to include a section on never merging your own PR, even if you're a maintainer..?

**Comment by m4v15** [comment]
@finnhodgkin 🙄  apparently it's needed I suppose...

**Review by Karyum — APPROVED** [review]

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -126,11 +132,20 @@ Make sure your PR has the following
...
     + Only maintainers should merge the PR's - if they are slacking, please feel free to give them a nudge in the comments of the PR.
+   6. **If your recipe is yet to be cooked:** Please give the PR a `Pending Cooking` label
+  
 
 ## Instructions for maintainers
 Once a pull request has been reviewed by 3 people **who ate the meal**, as well as or including a maintainer, it can be merged. A PR should never be merged if a reviewer has requested changes and not yet re-reviewed it.
 
 If a PR is up for an unreasonable amount of time without progress, maintainers should consult the creator on whether or not it will be finished at some point, and if not, close it.
+
+If a PR has the `Pending Cooking` label attached, it can remain open indefinitely.
```
Comment (matthewdking) [inline CONTRIBUTING.md:143]: I'm not sure about indefinitely? Maybe something the maintainers should use their best judgement on?

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -126,11 +132,20 @@ Make sure your PR has the following
...
+
+Maintainers should not merge their own PR's unless under extenuating circumstances and after discussion with others.
```
Comment (matthewdking) [inline CONTRIBUTING.md:145]: :laughing:

File: CONTRIBUTING.md (hunk 4 above)
Comment (m4v15) [reply CONTRIBUTING.md:143]: I'm not sure we can trust a maintainers best judgement...

**Comment by finnhodgkin** [comment]
Can we get this merged?

**Comment by m4v15** [comment]
@mattlub merge plz

**Pull request merged**